usaspending_api/common/helpers/fiscal_year_helpers.py

```diff
@@ -121,39 +121,57 @@ def dates_are_fiscal_year_bookends(start, end):
 def generate_fiscal_date_range(min_date: datetime, max_date: datetime, frequency: str) -> list:
     """
     Using a min date, max date, and a frequency indicator generates a list of dictionaries that contain
-    the fiscal year, fiscal quarter, and fiscal month.
+    the fiscal year, fiscal quarter, and fiscal month, or calendar year.
```

Comment:
I see understand what you're saying here. It will either return a dictionary with:
1. fiscal year, fiscal quarter, and fiscal month
OR
2. calendar year

But I didn't gain that understanding by reading the comment. I had to read through the code. Could you make this a little more clear. Maybe something like this:
```
Using a min date, max date, and a frequency indicator generates either a list of dictionaries
containing fiscal year information (fiscal year, fiscal quarter, and fiscal month) or a list 
of dictionaries containing calendar year information (calendar year).
```

Reply:
Your comment is pretty detailed and straightforward so I have used that in the file :)